lib/foodcritic/command_line.rb

```diff
@@ -132,9 +224,20 @@ def version
     #
     # @return [Boolean] True if the paths exist.
     def valid_paths?
+      tst = false
       paths = options[:cookbook_paths] + options[:role_paths] +
-        options[:environment_paths]
-      paths.any? && paths.all? { |path| File.exist?(path) }
+              options[:environment_paths]
+      paths.flatten!
+      if paths.any?
+        tst = true
+        paths.each do |path|
+          if !path || path.nil? || !File.exist?(File.expand_path(path))
```

Comment:
call paths.compact to drop the nil's, then you can drop the first two parts of this conditional